src/routes/safe/components/Balances/SendModal/screens/SendFunds/index.tsx

```diff
@@ -212,7 +214,7 @@ const SendFunds = ({
                   <Col>
                     <TokenSelectField
                       initialValue={selectedToken}
-                      isValid={tokenAddress && String(tokenAddress).toUpperCase() !== 'ETHER'}
+                      isValid={tokenAddress && String(tokenAddress).toUpperCase() !== nativeCoin.name.toUpperCase()}
```

Comment:
I know it's not related to this... but OMG, that `tokenAddress` compared to `'ETHER'` just blew my mind.